src/components/ReturnButton/ReturnButton.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Meta } from '@storybook/react';
+import { reactRouterParameters, withRouter } from 'storybook-addon-remix-react-router';
+import { ReturnButton } from './ReturnButton';
+
+export default {
+  title: 'Components/ReturnButton',
+  component: ReturnButton,
+  decorators: [withRouter]
+} as Meta;
+
+export const Default = {
+  parameters: {
+    reactRouter: reactRouterParameters({
+        routing: {
+          path: '/',
+          handle: 'Home Page',
+        }
+    })
+  }
+}
```

Comment:
In this case I followed the documentation to integrate react-router with storybook using decorators

[Link to Storybook Doc](https://storybook.js.org/addons/storybook-addon-react-router-v6)